core/src/main/java/org/apache/accumulo/core/file/rfile/SplitLarge.java

```diff
@@ -53,10 +56,25 @@
   }
 
   public static void main(String[] args) throws Exception {
+    new SplitLarge().execute(args);
+  }
+
+  @Override
+  public String keyword() {
+    return "split-large";
+  }
+
+  @Override
+  public String description() {
+    return "Splits an RFile into large and small key/value files";
+  }
+
+  @Override
+  public void execute(String[] args) throws Exception {
     Configuration conf = new Configuration();
     FileSystem fs = FileSystem.get(conf);
     Opts opts = new Opts();
-    opts.parseArgs(SplitLarge.class.getName(), args);
+    opts.parseArgs("accumulo split-large", args);
```

Comment:
I don't think prefixing this with "accumulo" is appropriate here, because starts including information about the script that launched this Java code, and I don't think the code should make such assumptions.

I suggest:
```suggestion
    opts.parseArgs(keyword(), args);
```

Reply:
Do you think it is ever appropriate? Taking a quick look it appears the following commands are prefixed with "accumulo" in the same way as this one: `convert-config`, `file-info`, `create-token`, `init`, `wal-info`.

Reply:
I just followed what was done in CreateToken 😀

Reply:
Seems to me that those are wrong, too, but it's fine if it's consistent. As far as I can tell, it's only used for printing the help/usage.

Reply:
> Seems to me that those are wrong, too, but it's fine if it's consistent.

It doesn't look like it is consistent. Looks like in some places the command is hardcoded (e.g., `opts.parseArgs("accumulo split-large"...`) while others use `.class.getName()` (e.g., `opts.parseArgs(SplitLarge.class.getName()...`). It might be easy to make these consistent and I could go ahead and do that if it seems like it would be worth it.

Reply:
The class name is appropriate for those with `main` methods that aren't `KeywordExecutable`. I think `keyword()` is probably most appropriate for those that are `KeywordExecutable`. But, making these consistent can be done as a follow-up so it doesn't hold this one up.